example/default.nix

```diff
@@ -0,0 +1,43 @@
+{ pkgs ? import <nixpkgs> { } }:
```

Comment:
If you remove the _picocli-gen_ artifact from the _pom.xml_; this actually builds!
So in turns of proof of concept it's great validation but the possible set of pom files to handle is huge :/